x/group/simulation/operations_test.go

```diff
@@ -215,6 +216,73 @@ func (suite *SimTestSuite) TestSimulateCreateProposal() {
 	suite.Require().Len(futureOperations, 0)
 }
 
+func (suite *SimTestSuite) TestWithdrawProposal() {
+	// setup 1 account
+	s := rand.NewSource(1)
+	r := rand.New(s)
+	accounts := suite.getTestingAccounts(r, 3)
+	acc := accounts[0]
+
+	// setup a group
+	ctx := sdk.WrapSDKContext(suite.ctx)
+	addr := acc.Address.String()
+	groupRes, err := suite.app.GroupKeeper.CreateGroup(ctx,
+		&group.MsgCreateGroup{
+			Admin: addr,
+			Members: []group.Member{
+				{
+					Address: addr,
+					Weight:  "1",
+				},
+			},
+		},
+	)
+	suite.Require().NoError(err)
+
+	// setup a group account
```

Comment:
```suggestion
	// setup a group policy
```

could you also update the rest of the comments that might still mention "group account" in this file?